src/js/common/schemaform/review/ArrayField.jsx

```diff
@@ -201,11 +208,10 @@ class ArrayField extends React.Component {
                 <div className="row small-collapse">
                   <SchemaForm
                       reviewMode
-                      pageData={item}
-                      schema={this.getItemSchema(index)}
+                      data={item}
+                      schema={itemSchema}
                       uiSchema={arrayPageConfig.uiSchema}
-                      title={pageTitle}
-                      hideTitle
+                      title={itemSchema.title || ''}
```

Comment:
I did it this way because as-is, it added the title only because it was required, but always hid it. This title will be the one next to the edit button, so I don't know that it'll make much sense to have the page title on the item we're editing.